generators/server/templates/_pom.xml

```diff
@@ -1203,37 +1203,12 @@
             </dependencies>
             <build>
                 <plugins>
-                    <%_ if (!skipClient) { _%>
+                    <%_ if (!skipClient && clientFw === 'angular2') { _%>
```

Comment:
probably to make the possibility to use another thing than angular in the future